### PR TITLE
CLOUDP-174225: Add example for metrics command

### PR DESCRIPTION
**Pull request by gssbzn**
## Proposed changes

_Jira ticket:_ CLOUDP-174225

Docs improvements based on internal feedback

### CODE REVIEW

File: docs/mongocli/command/mongocli-ops-manager-metrics-databases-describe.txt [hunk 1]
```diff
@@ -112,3 +112,10 @@ Inherited Options
      - false
      - Human-readable label that identifies the profile to use from your configuration file. To learn about profiles for the Atlas CLI, see https://dochub.mongodb.org/core/atlas-cli-save-connection-settings. To learn about profiles for MongoCLI, see https://dochub.mongodb.org/core/atlas-cli-configuration-file.
 
+Examples
+--------
+
+.. code-block::
+
+   # List metrics for the database test of the process e4ac1e57c58cc9c8aaa5a1163a851993
+ mongocli ops-manager metrics disk describe e4ac1e57c58cc9c8aaa5a1163a851993 test --period P1DT12H --granularity PT5
```
Comment (swattyT) [inline docs/mongocli/command/mongocli-ops-manager-metrics-databases-describe.txt:121]: mongocli ops-manager metrics `database` describe

File: docs/mongocli/command/mongocli-cloud-manager-metrics-databases-describe.txt [hunk 2]
```diff
@@ -112,3 +112,10 @@ Inherited Options
      - false
      - Human-readable label that identifies the profile to use from your configuration file. To learn about profiles for the Atlas CLI, see https://dochub.mongodb.org/core/atlas-cli-save-connection-settings. To learn about profiles for MongoCLI, see https://dochub.mongodb.org/core/atlas-cli-configuration-file.
 
+Examples
+--------
+
+.. code-block::
+
+   # List metrics for the database test of the process e4ac1e57c58cc9c8aaa5a1163a851993
+ mongocli ops-manager metrics disk describe e4ac1e57c58cc9c8aaa5a1163a851993 test --period P1DT12H --granularity PT5
```
Comment (swattyT) [inline docs/mongocli/command/mongocli-cloud-manager-metrics-databases-describe.txt:121]: ps-manager metrics `database` describe

File: docs/mongocli/command/mongocli-ops-manager-metrics-databases-describe.txt (hunk 1 above)
Comment (gssbzn) [reply docs/mongocli/command/mongocli-ops-manager-metrics-databases-describe.txt]: good catch

**Review by swattyT — APPROVED** [review]
LGTM!

File: internal/cli/opsmanager/metrics/process.go [hunk 3]
```diff
@@ -64,9 +64,11 @@ func ProcessBuilder() *cobra.Command {
 		Use:     "process <hostId>",
 		Short:   "Get measurements for a given host.",
 		Aliases: []string{"processes"},
-		Args:    require.ExactArgs(1),
+		Example: `# List metrics for the process e4ac1e57c58cc9c8aaa5a1163a851993
+mongocli ops-manager metrics process e4ac1e57c58cc9c8aaa5a1163a851993 --period P1DT12H --granularity PT5`,
```
Comment (sarahsimpers) [inline internal/cli/opsmanager/metrics/process.go:68]:
```suggestion
  mongocli ops-manager metrics process e4ac1e57c58cc9c8aaa5a1163a851993 --period P1DT12H --granularity PT5`,
```

**Review by sarahsimpers — COMMENTED** [review]
Need to indent 2 spaces

**Comment by gssbzn** [comment]
@sarahsimpers RFAL

**Review by sarahsimpers — APPROVED** [review]
LGTM

**Pull request merged**